src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.Tag;
+
+/**
+ * Represents the relationship of tagging between a Person and a Tag
+ *
+ */
+public class Tagging {
+    private boolean isAdded;//if not isAdded, tag is deleted
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.